x/concentrated-liquidity/incentives_test.go

```diff
@@ -0,0 +1,119 @@
+package concentrated_liquidity_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	cl "github.com/osmosis-labs/osmosis/v14/x/concentrated-liquidity"
+	"github.com/osmosis-labs/osmosis/v14/x/concentrated-liquidity/types"
+)
+
+var defaultPoolId = uint64(1)
+
+func (s *KeeperTestSuite) TestCreateAndGetUptimeAccumulators() {
+	// We expect there to be len(types.SupportedUptimes) number of initialized accumulators
+	// for a successful pool creation. We calculate this upfront to ensure test compatibility
+	// if the uptimes we support ever change.
+	curExpectedAccumValues := []sdk.DecCoins{}
+	for range types.SupportedUptimes {
+		curExpectedAccumValues = append(curExpectedAccumValues, cl.EmptyCoins)
+	}
+	s.Require().Equal(len(types.SupportedUptimes), len(curExpectedAccumValues))
+
+	type initUptimeAccumTest struct {
+		poolId              uint64
+		initializePool      bool
```

Comment:
nit: maybe we should call this `initializePoolAccum` since we dont actually initialize a pool in the test.

Reply:
Nice catch, naming was from an earlier version of tests. Changing!